hibernate/hibernate-orm/src/test/java/io/quarkus/qe/hibernate/AbstractHQLConsoleIT.java

```diff
@@ -0,0 +1,131 @@
+package io.quarkus.qe.hibernate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import io.quarkus.qe.hibernate.hql.HQLConsoleClient;
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+
+@Tag("QUARKUS-6243")
+public abstract class AbstractHQLConsoleIT {
+
+    private static HQLConsoleClient hqlClient;
+    private static final int DEFAULT_PAGE_SIZE = 15;
+    private static final String PERSISTENCE_UNIT = "test-hql-pu";
+
+    @BeforeEach
+    public void beforeEach() {
+        hqlClient = new HQLConsoleClient(getApp().getURI(Protocol.HTTP).toString(), PERSISTENCE_UNIT);
```

Comment:
`HQLConsoleClient` has test class scope, so I don't understand why you create it per each test method. But if you want to do that for example to clear previous results, it is fine. It feels like a strange pattern:

1. store `hqlClient` in static variable
2. only use it in a member methods
3. create the `hqlClient` in a `beforeEach` but it only is used within one test method

I think the proper pattern would be that inside one method you use try-with-resources and close Vert.x/HttpClient with it. I don't wan to be annoying, so just drop the `static` and I can live with it.